src/components/ConnectWallet/ConnectWalletModal.tsx

```diff
@@ -73,10 +73,22 @@ export const ConnectWalletModal: FC<ModalBaseProps> = ({ isOpen, onClose }) => {
 
       {error && <div className="mt-6 text-center">{error.message}</div>}
 
-      <div className="mt-6 text-center font-mono text-xs">
-        By connecting a wallet, you agree to XXXX Terms of Service and
-        acknowledge that you have read and understand the XXXX Protocol
-        Disclaimer
+      <div className="mt-6 text-justify font-mono text-xs">
+        By connecting your wallet to Fortress finance, you acknowledge that you
+        have read and understand the{" "}
+        <Link
+          href="https://docs.fortress.finance/protocol"
```

Comment:
We should open these links in a new tab.

Reply:
Agreed, will changed it.